hooks/post_gen_project.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+
+def remove(filepath):
+    if os.path.isfile(filepath):
+        os.remove(filepath)
+
+
+if "{{cookiecutter.notebooks_in_analysis}}" == "n":
+    remove("analysis/01_analysis_step_a.md")
+    remove("analysis/02_analysis_step_b.md")

```

Comment:
Now this is very much details that is not necessary imo, but if `notebooks_in_analysis` == 'y', you could possibly remove the .deleteme file in `analysis`. But up to you if you feel that is appropriate or adds unnecessary code.